site/_extensions/cookbook_gallery_generator.py

```diff
@@ -10,21 +10,16 @@ def main(app):
 
     title = "Cookbooks Gallery"
 
-    subtext = ""
```

Comment:
Just wondering what's the motivation for removing this and putting the subtext in as HTML?

Reply:
I can put that back, it was an extraneous edit made in the beginning of wrestling with the gallery generator being broken. I was probably trying to track down a broken extension and never undid it.

Reply:
That makes sense. Yes, if you can revert that part back, I think it will be easier to maintain the subtext content if it stays in its own markdown file.